dtao.c

```diff
@@ -345,13 +398,138 @@ static struct zwlr_layer_surface_v1_listener layer_surface_listener = {
 	.closed = layer_surface_closed,
 };
 
+struct input_state {
+	struct wl_surface *surface;
+	double x, y;
+	uint32_t button;
+};
+
+static void
+noop() {}
```

Comment:
In the other reviews I left this alone, but I don't like functions without strict prototypes, and I think @djpohly doesn't either.